### PR TITLE
Add tabs to the categories view

### LINKED ISSUE #30: Make category view the same as project view
The 'project' view has been updated to use tabs due to the number of technologies that are displayed.  The categories should be updated to the use tabs too.

### PR DESCRIPTION
**Pull request by adrianz-kainos**
Resolves #30

### CODE REVIEW

File: views/pages/categoryRadar.ejs [hunk 1]
```diff
@@ -40,38 +40,73 @@
...
+                    var heading = "";
+                    var lastTechId = -1;
+                    var lastTechNum=-1;
+
+                    for (var i = 0, len = technologies.length; i < len; i++) {
+                        var technology = technologies[ i ];
+                    if( technology.status != heading) {
+                        heading = technology.status;
+
+                    if(i != 0) { %>
```
Comment (xjedam) [inline views/pages/categoryRadar.ejs]: Very convoluted logic, is it possible to simplify/comment on some of the code here?

File: package.json [hunk 2]
```diff
@@ -22,33 +22,34 @@
...
     "passport-local": "1.0.0",
+    "pg": "6.1.0",
+    "pg-large-object": "latest",
     "sanitize-html": "latest",
-    "db-migrate": "0.10.0-beta.12",
-    "db-migrate-pg": "latest",
-    "express-validator" : "latest",
-    "connect-flash" : "0.1.1"
+    "sinon": "latest",
+    "underscore": "^1.8.3"
```
Comment (adrianz-kainos) [inline package.json:52]: FYI: I only added the underscore dependency. The remaining changes were made by npm auto-sorting.

File: handlers/web/categoriesWebHandler.js [hunk 3]
```diff
@@ -26,8 +27,24 @@ CategoriesWebHandler.technologiesForCategory = function (req, res) {
...
+                };
+            }).value();
+
         var category = cache.getCategory(cname);
-        res.render('pages/categoryRadar', {category: category, technologies: values, user: req.user});
+
+        res.render('pages/categoryRadar', {
+            category: category,
+            technologies: values, // used by radar.js
+            technologiesInGroups: technologiesInGroups,
```
Comment (adrianz-kainos) [inline handlers/web/categoriesWebHandler.js:45]: `technologiesInGroups` and  `technologies` contain the same data (structured differently). I couldn't get rid of `technologies` without reworking radar.js. Changing radar.js to use `technologiesInGroups` might take a whole workday. Is it worth it?

File: views/pages/categoryRadar.ejs [hunk 4]
```diff
@@ -5,19 +5,16 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <meta name="description" content="">
```
Comment (adrianz-kainos) [inline views/pages/categoryRadar.ejs:8]: `meta name="description"` is in `head.ejs`

File: package.json (hunk 2 above)
Comment (xjedam) [reply package.json:52]: 👍

File: handlers/web/categoriesWebHandler.js (hunk 3 above)
Comment (xjedam) [reply handlers/web/categoriesWebHandler.js:45]: It looks great now :)

**Pull request merged**